### DISCUSSION TITLE
Issues with GitHub Education Application Approval

### DISCUSSION THREAD

**Discussion by ramjanfaragx** [community/community#58052] [GitHub Education]
### Hi everyone,

Dear GitHub Education Team,

Hello! I am writing to inquire about an issue regarding my GitHub Education application. I have attempted to apply for GitHub Education multiple times, but each attempt has been unsuccessful. I would like to understand the reasons behind this and seek a solution.

Firstly, I possess an edu email address from my school, and I have tried submitting my application by taking photos with my camera and uploading images. I am confident that the images I uploaded are in English, clearly visible, and include my enrollment and graduation dates as well as my name. However, my application still fails to be approved.

I am eager to join GitHub Education, as it would provide me with immense convenience in learning programming and developing projects. May I ask you what reasons could have caused my application to be denied? Are there any solutions that can help me successfully apply?

Thank you once again for your attention and assistance. I look forward to your response!

Best regards

>

**Comment by ramjanfaragx** [comment]
The text in the photo you used was unable to be read. If using your device camera, make sure the image is clear and legible. For some languages, you may need to capture the original document beside an English translation.

You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

Your document does not appear to include a date demonstrating current academic affiliation. For countries utilizing non-standard calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible.

This is the failure message

**Comment by svadrutk** [comment]
Yeah I was trying to reverify and had the same problem
